spec/unit/application_spec.rb

```diff
@@ -473,6 +473,33 @@ def configure_proxy_environment_variables_stubs
     end
   end
 
+  describe 'run_chef_client' do
+    before do
+      @app = Chef::Application.new
+      @app.stub(:fork_chef_client)
+      @app.stub(:run_with_graceful_exit_option)
+    end
```

Comment:
Could you update this section to use the RSpec 3 patterns we've been moving towards?

* Use `let(:var)` instead of `@var` to define test variables
* Use `:allow` or `:expect` instead of `:stub`
* Use `:expect/to` instead of `:should` in test blocks